### PR TITLE
[Platform]: PPP Project table header and icon changes

### PR DESCRIPTION
**Pull request by chinmehta**
# [Platform]: PPP Project table header and icon changes

## PPP Project table header text changes from "Integrates into PPP" to "Currently integrates into PPP", changed icons

**Issue:** # [Issue-2900](https://github.com/opentargets/issues/issues/2900)
**Deploy preview:** [PPP Preview](https://deploy-preview-107--ot-platform.netlify.app/projects)

## Type of change

* Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

* Test A: Text on table header is "Integrates into PPP", ppp documentation link on projects page redirects to [platform-docs](https://platform-docs.opentargets.org/partner-preview-platform).
* Test B: Text on table header is "Currently integrates into PPP", ppp documentation link on projects page redirects to [ot-home/ppp-doc](http://home.opentargets.org/ppp-documentation).

## Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by LucaFumis — APPROVED** [review]
Thank you @chinmehta - code looks good. 
I understand icons will need final approval and changes in index.html (profile loading) will have to be reverted before merging.

**Comment by buniello** [comment]
Thank you Chintan, the table looks good. We just need to change the "Diseases Mapped" header into "Disease Mapped in the PPP"
<details><summary>Browser metadata</summary>





```
Path:      /projects
Browser:   Chrome 110.0.0.0 on Mac OS 10.15.7
Viewport:  1433 x 707 @2x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-107--ot-platform.netlify.app%2Fprojects&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=110.0&resolution=1433x707&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-107--ot-platform.netlify.app/projects) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6410abd3536ca300086f8cfa&commentID=641828f90327f117ac07b8e0&resolution=resolved)

**Pull request merged**
